participants/Gregorowicz/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,10 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+
+def isprime(num):
+    for n in range(2,int(num**0.5)+1):
```

Comment:
Perhaps you could consider adding some edge cases like 0 or 1 to return False before this loop.